### ISSUE TITLE
Can't build against ghc-8.4.2/base-4.11.1.0

### ISSUE THREAD

**Issue by clumens** [ozataman/csv-conduit#37]
This is the Semigroup thing:

```[3 of 7] Compiling Data.CSV.Conduit.Conversion ( src/Data/CSV/Conduit/Conversion.hs, dist/build/Data/CSV/Conduit/Conversion.o )

src/Data/CSV/Conduit/Conversion.hs:781:10: error:
    â¢ No instance for (Semigroup (Parser a))
        arising from the superclasses of an instance declaration
      There are instances for similar types:
        instance Semigroup (Data.Attoparsec.Internal.Types.Parser i a)
          -- Defined in âData.Attoparsec.Internal.Typesâ
    â¢ In the instance declaration for âMonoid (Parser a)â
    |
781 | instance Monoid.Monoid (Parser a) where
    |          ^^^^^^^^^^^^^^^^^^^^^^^^
```

I could probably work on a patch, given enough time, but I'd have to get familiar with this stuff first.  I assume you'll want to keep compatibility with older versions of base, too.

**Comment by MichaelXavier** [comment]
This is released with `csv-conduit-0.7.0.0`

**Comment by clumens** [comment]
Thanks!
